bot/client.js

```diff
@@ -57,4 +57,9 @@ client.on('message', (message) => {
   }
 });
 
+// Send message for a new user
+client.on('guildMemberAdd', (member) => {
+  member.sendMessage(`Welcome to the server ${member.user.username}!. Enjoy your stay!`);
```

Comment:
Consider that your implementation solves the problem, and elegantly, but falls just short in that your solution ​within the client.js file does not implement the controller you created. So in effect, any further maintenance will require duplicate efforts in keeping both messages identical.